src/Umbraco.Cms.Integrations.Crm.Dynamics/Models/DynamicsModule.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Umbraco.Cms.Integrations.Crm.Dynamics.Models
+{
+    public enum DynamicsModule
```

Comment:
You could look at a "Flags" enum here, and then use "HasFlag" in your checks to see what's been selected.  Explanation [here](https://dev.to/ayodejii/enum-flags-in-c-20a6).

Reply:
Updated